src/psyclone/psyir/nodes/operation.py

```diff
@@ -203,6 +204,13 @@ def datatype(self):
         '''
         return self.children[0].datatype
 
+    @property
+    def operands(self) -> DataNode:
```

Comment:
Should this return a tuple of 1-element to match the other "operands"? otherwise we would need to do an isinstance before calling it for each generic operation.

Reply:
@arporter opinions?

Reply:
From a quickish grep of the code, the code that handles UnaryOperations seems to be distinct from that for BinaryOperations and therefore it's probably best to have `operand` and `operands` as that will avoid the tuple-unpacking error that could result if we try to do the `lhs, rhs = bop.operands` when bop is actually unary. I think we probably have to try it and see.

Reply:
Ah I missed this converation. I'll switch it to operand now.